### PR TITLE
chore: ignore discriminator error

### PR DESCRIPTION
**Pull request by AntonKozachuk**
## What/Why/How?
We would like to temporarily ignore an error regarding the discriminator usage.

## Reference
N/a

## Testing
Locally.

## Screenshots (optional)
N/a

## Check yourself

- [x] Code is linted
- [ ] Tested with redoc/reference-docs/workflows
- [ ] All new/updated code is covered with tests

## Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

**Review by RomanHotsiy — CHANGES_REQUESTED** [review]
Any tests?

File: packages/core/src/rules/utils.ts [hunk 1]
```diff
@@ -118,11 +118,14 @@ export function validateExample(
       }
     }
   } catch (e) {
-    report({
-      message: `Example validation errored: ${e.message}.`,
-      location: location.child('schema'),
-      from: location,
-    });
+    const discriminatorError = 'discriminator: requires oneOf or anyOf composite keyword';
+    if (e.message !== discriminatorError) {
```
Comment (tatomyr) [inline packages/core/src/rules/utils.ts:122]: What about `if (e.message === discriminatorError) { return }` ?

**Review by tatomyr — APPROVED** [review]

File: packages/core/src/rules/oas3/__tests__/no-invalid-media-type-examples.test.ts [hunk 2]
```diff
@@ -438,4 +438,36 @@ describe('no-invalid-media-type-examples', () => {
       ]
     `);
   });
+
+  it('should not report if allOf used with discriminator', async () => {
```
Comment (RomanHotsiy) [inline packages/core/src/rules/oas3/__tests__/no-invalid-media-type-examples.test.ts:442]: This test needs to be improved.

**Review by RomanHotsiy — APPROVED** [review]
The test need to be improved but let's merge and release it and then fix the test.

**Pull request merged**